spec.html

```diff
@@ -881,6 +892,42 @@ <h1>ToElementDescriptors ( _elementObjects_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-to-decorator-property-descriptor" aoid=ToDecoratorPropertyDescriptor>
+      <h1>ToDecoratorPropertyDescriptor ( _elementObject_ )</h1>
+      <emu-alg>
+        1. Let _desc_ be a new Property Descriptor that initially has no fields.
+        1. Let _hasEnumerable_ be ? HasProperty(_Obj_, `"enumerable"`).
+        1. If _hasEnumerable_ is *true*, then
+          1. Let _enumerable_ be ToBoolean(? Get(_Obj_, `"enumerable"`)).
+          1. Set _desc_.[[Enumerable]] to _enumerable_.
+        1. Let _hasConfigurable_ be ? HasProperty(_Obj_, `"configurable"`).
+        1. If _hasConfigurable_ is *true*, then
+          1. Let _configurable_ be ToBoolean(? Get(_Obj_, `"configurable"`)).
+          1. Set _desc_.[[Configurable]] to _configurable_.
+        1. Let _hasValue_ be ? HasProperty(_Obj_, `"value"`).
+        1. If _hasValue_ is *true*, then
+          1. Let _value_ be ? Get(_Obj_, `"method"`).
```

Comment:
What do you think about throwing if it isn't callable, like we do for get and set?

Reply:
Oh yeah, I meant to include that and forgot, fixed.